tutorials/containers/microk8s-clustering

```diff
@@ -0,0 +1,71 @@
+---
+id: raspberrypi-clustering-instructions
+summary: Create a raspberry pi cluser using microk8s
+categories: clusters
+tags: kubernetes, beginner, multipass, container, microk8s
+difficulty: 3
+status: draft
+published: 2019-09-13
+author: Ammar Naqvi <ammar.naqvi@canonical.com>, Rhys Davies <rhys.davies@canonical.com>
+feedback_url: https://github.com/ubuntu/microk8s/issues/
+
+---
+
+Canonical is dedicated to enhancing the way we build and deliver technology through our continued investment in innovation and the open-source community. Over the past year, we’ve worked hard to realize the next phase of scaling and deployments on Kubernetes; we introduced [MicroK8s][microk8s], the lightweight, fast, powerful Kubernetes up in 60 seconds.
+
+
+We’ve heard from our users, partners, and customers and today, we’re pleased and excited to announce multi-node clustering in MicroK8s. You can now join multiple deployments to form a cluster. Welcome to simple, enterprise-grade Kubernetes! 
+
+# Test-Drive Time
+
+A preview release is available for you to test and give us your feedback. Lets set up the MicroK8s cluster.
+On two or more machines install MicroK8s from the 1.15/edge/clustering channel:
+
+```bash
+sudo snap install microk8s --classic --channel=1.15/edge/clustering
+```
+
+One of these machines need to be assigned as the master, hosting the control plane. On the master machine, run the following command for each node you want to add:
+
+```bash
+sudo microk8s.add-node
+```
+
+The microk8s.add-node will generate a connection string in the form of <master_ip>:<port>/<token> and will prompt you to use the microk8s.join command from the node you need to add to the cluster. 
```

Comment:
You may want to add some network considerations to requirements: all Pis need to be on the same network, and you need to have terminal windows ready to connect to each through SSH.

Reply:
Added this to the "what you'll need" section. Are there tutorials to link to that walk through connecting pis through SSH?